infra/terraform/test-org/org/locals.tf

```diff
@@ -113,7 +113,7 @@ locals {
       short_name  = "ecommerce-microservices"
       org         = "GoogleCloudPlatform"
       description = "Deploys a web-based ecommerce app into a multi-cluster Google Kubernetes Engine setup."
-      owners      = ["NimJay"]
+      owners      = ["@GoogleCloudPlatform/dee-platform-ops"]
```

Comment:
Several of the downstream modules expect these values to be users, might need to list out for now.